Cargo.toml

```diff
@@ -6,23 +6,24 @@ members = [
 
 # The release profile, used for `cargo build`.
 [profile.dev]
-incremental = true
-opt-level = 0
-debug = true
-rpath = false
-lto = false
-debug-assertions = true
-overflow-checks = true
-panic = 'unwind'
+# Enables thin local LTO and some optimizations.
+opt-level = 1
```

Comment:
I’m not sure for dev we want any optimisations, also setting 1 turns off the debug asserts which I think we’re using to get better feedback if something is wrong on dev

Reply:
I was trying to find where it said that this would disable debug assertions, but I couldn't find it.